test-cases/longevity/longevity-100gb-4h.yaml

```diff
@@ -6,15 +6,19 @@ stress_cmd: ["cassandra-stress write cl=QUORUM duration=240m -schema 'replicatio
              "cassandra-stress read  cl=QUORUM duration=240m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
              ]
 
-n_db_nodes: 6
-n_loaders: 2
+n_db_nodes: 5
+n_db_zero_token_nodes: 1
```

Comment:
What is the difference between this `n_db_zero_token_nodes` and `num_zero_token_nodes`?

Reply:
left only n_db_zero_token_nodes